From fed80244f352ec98afd04595f4784ff60fea65d3
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Wed, 24 Feb 2010 15:30:09 +0800
Subject: kmemcheck fix compiling for 2.6.27 

This patch intended to fix compiling error for 2.6.27
when back porting.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/x86/mm/kmemcheck/error.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/x86/mm/kmemcheck/error.c b/arch/x86/mm/kmemcheck/error.c
index 4901d0d..2fca45e 100644
--- a/arch/x86/mm/kmemcheck/error.c
+++ b/arch/x86/mm/kmemcheck/error.c
@@ -133,7 +133,11 @@ void kmemcheck_error_recall(void)
 		break;
 	}
 
-	__show_regs(&e->regs, 1);
+#ifdef CONFIG_X86_64
+	__show_regs(&e->regs);
+#else
+	__show_registers(&e->regs,1);
+#endif
 	print_stack_trace(&e->trace, 0);
 }
 
-- 
1.6.3.3

